README.md

```diff
@@ -88,6 +86,42 @@ You will find a **server-rendering** example in the repo´s example directory.
 
 redux-router will notice if the router state in your Redux store changes from an external source other than the router itself — e.g. the Redux Devtools — and trigger a transition accordingly!
 
+## Differences with React-Router-Redux
+
+#### React-Router-Redux
```

Comment:
We normally brand it with all lowercase: react-router-redux. And "React Router" doesn't normally include a dash, but the titlecase works.